src/main/steps/respondent/yourhearings/hearings/content.ts

```diff
@@ -1,10 +1,19 @@
+import { applyParms } from '../../../../steps/common/url-parser';
 import { TranslationFn } from '../../../../app/controller/GetController';
 import { generateContent as yourhearingshearingscontent } from '../../../common/yourhearings/hearings/content';
+import {  RESPONDENT_TASK_LIST_URL } from '../../../../steps/urls';
 
 export { form } from '../../../common/yourhearings/hearings/content';
 export const generateContent: TranslationFn = content => {
   const hearingsContent = yourhearingshearingscontent(content);
+  hearingsContent.linkforsupport = '/respondent/support-you-need-during-case/attending-the-court';
+  const request = content.additionalData?.req;
+  const caseData = request.session.userCase;
   return {
     ...hearingsContent,
+    breadcrumb:{
```

Comment:
Breadcrumb shd be shown only for CA applicant journey and for all other journey it should be just back navigation.

Reply:
Yes this has been taken care now.